backend/framework/qlf/dashboard/bokeh/qagetbias/main.py

```diff
@@ -155,10 +172,10 @@ def load_qa(self):
 
         from dashboard.bokeh.qlf_plot import html_table
 
-        print(tests['getbias']['BIAS_AMP_REF'])
-        nrg= tests['getbias']['BIAS_AMP_NORMAL_RANGE']
-        wrg= tests['getbias']['BIAS_AMP_WARN_RANGE']
-        #names=['BIAS'], vals=[ getbias['BIAS_AMP']],
+        
+        # nrg= tests['getbias']['BIAS_NORMAL_RANGE']
```

Comment:
If you are not going to use it just delete it. We can recover code from git history